client/components/Username.vue

```diff
@@ -1,6 +1,7 @@
 <template>
 	<span
-		:class="['user', {[nickColor]: store.state.settings.coloredNicks}, {active: active}]"
+		:class="['user ', {[nickColor]: store.state.settings.coloredNicks}, {active: active}]"
```

Comment:
why did you add a space here?

Reply:
This was a goof-up on my side, apparently I didn't remove it when I moved the user modes to data-mode attribute. Will fix!